### PR TITLE
InterSatellite Sync CLI and UI stubs

### PR DESCRIPTION
**Pull request by jyejare**
InterSatellite Sync UI and CLI stubs.

This covers most of the scenarios for ISS excluding Redhat content sync.
The stubs for Redhat contents inter satellite sync will be added in next PR.

### CODE REVIEW

File: tests/foreman/ui/test_satellitesync.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+class InterSatelliteSyncTestCase(UITestCase):
+    """Implements InterSatellite Sync tests in UI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_show_repo_export_history(self):
+        """
```
Comment (oshtaier) [inline tests/foreman/ui/test_satellitesync.py]: Probably you forgot description here

File: tests/foreman/ui/test_satellitesync.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        user and time.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_show_cv_export_history(self):
+        """
```
Comment (oshtaier) [inline tests/foreman/ui/test_satellitesync.py]: And here

File: tests/foreman/cli/test_satellitesync.py [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+        @steps:
+
+        1. In upstream, Export CV version contents to a directory specified in
+        settings.
+        2. In downstream, Import these copied contents from some other
+        org/satellite.
+        3. In upstream, Add new packages to the CV.
+        4. Export the CV incrementally from the last date time.
+        5. In downstream, Import the CV incrementally.
+
```
Comment (sthirugn) [inline tests/foreman/cli/test_satellitesync.py]: What does upstream/downstream mean here?

File: tests/foreman/ui/test_satellitesync.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        2. The imported repo is enabled and sync.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_update_cdn_url(self):
+        """Update false CDN URL to import from upstream.
```
Comment (sthirugn) [inline tests/foreman/ui/test_satellitesync.py]: s/false/invalid

File: tests/foreman/cli/test_satellitesync.py [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+        1. Repo has been exported to iso successfully.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_negative_export_repo_iso_from_feature_datetime(self):
```
Comment (sthirugn) [inline tests/foreman/cli/test_satellitesync.py]: missed docstring

**Comment by sthirugn** [comment]
Pending comments

**Comment by sthirugn** [comment]
travis already caught the missed docstring test :)

File: tests/foreman/cli/test_satellitesync.py (hunk 3 above)
Comment (jyejare) [inline tests/foreman/cli/test_satellitesync.py]: Upstream : The satellite/org which exports the CV/repo to be imported by downstream satellite/org.
Downstream: The satellite/org which imports the exported CV/repo from upstream satellite.

**Comment by oshtaier** [comment]
ACK

**Comment by sthirugn** [comment]
ACK

**Pull request merged**